app/models/pet_application.rb

```diff
@@ -0,0 +1,4 @@
+class PetApplication < ApplicationRecord
+  belongs_to :pet
+  belongs_to :application
```

Comment:
Great job creating the association. You never utilized this by creating a join controller.